service/src/main/java/tds/config/web/resources/TaCheckinTimeResource.java

```diff
@@ -0,0 +1,16 @@
+package tds.config.web.resources;
+
+/**
+ * A HATEOAS representation of the TA Check-In Time value.
+ */
+public class TaCheckinTimeResource {
```

Comment:
Yep - to make sure a JSON object is returned that is consistent w/the way we've been returning other payloads.  Overkill?  Probably.